### PR TITLE
Update Nintendo - GameCube.json

### PR DESCRIPTION
**Pull request by rufotheone-fr**

### CODE REVIEW

File: clonelists/Nintendo - GameCube.json [hunk 1]
```diff
@@ -450,6 +453,8 @@
 		],
 		"Nickelodeon SpongeBob SquarePants - The Movie": [
 			["Nickelodeon SpongeBob Schwammkopf - Der Film", 1]
+			["2 Games in 1 - Bob L'eponge - Le Film + Tak 2 - Le Sceptre des Reves (France) (Disc 1) (Bob L'eponge - Le Film)", 1]
+			["2 Games in 1 - Nickelodeon SpongeBob Schwammkopf - Der Film + Nickelodeon SpongeBob Schwammkopf - Schlacht um Bikini Bottom (Germany) (Disc 1)", 1]
```
Comment (unexpectedpanda) [inline clonelists/Nintendo - GameCube.json:457]: For this to work, you need use [short names](https://github.com/unexpectedpanda/retool/wiki/Terminology#short-names). For the titles you've added, this means removing the countries.

**Review by unexpectedpanda — CHANGES_REQUESTED** [review]

File: clonelists/Nintendo - GameCube.json [hunk 2]
```diff
@@ -450,6 +453,8 @@
 		],
 		"Nickelodeon SpongeBob SquarePants - The Movie": [
 			["Nickelodeon SpongeBob Schwammkopf - Der Film", 1]
```
Comment (unexpectedpanda) [inline clonelists/Nintendo - GameCube.json:455]: `Nickelodeon SpongeBob Schwammkopf - Der Film` should have a priority of `2`, so Retool knows which German title to pick first.

We want both `2 Games in 1 - Nickelodeon SpongeBob Schwammkopf - Der Film + Nickelodeon SpongeBob Schwammkopf - Schlacht um Bikini Bottom (Disc 1)` and `(Disc 2)` to become the chosen German titles in each of their respective situations, so people don't complain that there are missing discs.

**Review by unexpectedpanda — CHANGES_REQUESTED** [review]

File: clonelists/Nintendo - GameCube.json (hunk 1 above)
Comment (rufotheone-fr) [reply clonelists/Nintendo - GameCube.json:457]: Got it!

Comment (unexpectedpanda) [reply clonelists/Nintendo - GameCube.json:457]: Feel free to update the commit, alternatively I can roll the changes separately into the next update :)

Comment (rufotheone-fr) [reply clonelists/Nintendo - GameCube.json:457]: Hi! I'm a little busy and I'm waiting for a massive dl to finish so I can test and append fixes.

Comment (unexpectedpanda) [reply clonelists/Nintendo - GameCube.json:457]: No worries -- I'll close out this PR and integrate the corrections directly :)

**Pull request closed without merging**